framework/src/modules/base_module.ts

```diff
@@ -39,7 +39,7 @@ export interface ModuleMetadata {
 	endpoints: {
 		name: string;
 		request?: Schema;
-		response: Schema;
+		response?: Schema;
```

Comment:
i found out that response is always expected in ABI so this actually doesn't work, and produces message
> The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received undefined"

either we always return something, or update `abi_handler.ts` to handle void response case